Automattic/wp-calypso
# Hosting Command Palette: Add breadcrumbs for nested commands above the input on Calypso

Related to https://github.com/Automattic/dotcom-forge/issues/4638

## Proposed Changes

This PR:

* adds breadcrumbs to the command palette for nested commands
* makes the style of the bread crumbs consistent with Figma design:

2aegKcEt9nAj7wBQ4yV5n4-fi-800_14498

**Before**:

<img width="1269" alt="Screenshot 2023-11-27 at 3 02 28 PM" src="https://github.com/Automattic/wp-calypso/assets/25575134/541896f1-b292-4bec-8548-1a5bb6f5e6c4">

**After**:

<img width="1253" alt="Screenshot 2023-11-27 at 3 02 36 PM" src="https://github.com/Automattic/wp-calypso/assets/25575134/e6d18a61-e257-47c4-9eb1-b888a58c7c11">

## Testing Instructions

* Navigate to Calypso Live Link or pull the changes from this branch locally
* Go to `/sites`
* Press `cmd + k` on macOS or `ctrl + k` on Windows
* Observe the command palette to appear
* Select one of the commands that has a nested command e.g. `Manage DNS`
* Observe that you can see breadcrumbs on the top

**Notes**;

* I see that we added a placeholder for `Search for a site` on Friday to nested commands but Figma design does not have it. Should we keep it or remove it?

Here is side by side comparison:

<img width="1595" alt="Screenshot 2023-11-27 at 3 07 35 PM" src="https://github.com/Automattic/wp-calypso/assets/25575134/daf8fc75-e0b3-4f96-b253-ce5e15eb3cf7">

## Pre-merge Checklist
